plugins/obs-ffmpeg/texture-amf.cpp

```diff
@@ -72,6 +72,77 @@ static AMFTrace *amf_trace = nullptr;
 static HMODULE amf_module = nullptr;
 static uint64_t amf_version = 0;
 
+/* ================================================================================================================= */
+/* The structure and tables below are used to determine the appropriate minimum encoding level for the codecs. AMF
+ * defaults to the highest level for each codec (AVC, HEVC, AV1), and some client devices will reject playback if the
+ * codec level is higher than its decode abilities, which make sense.
```

Comment:
```suggestion
 * codec level is higher than its decode abilities, which makes sense.
```

Or just remove `, which makes sense` entirely.